Project #2

tyao123 opened the pull request.

tyao123 committed:
Renamed folder name to include all members

TaeyoonLee committed:
added daycare datasets and merged datasets

TaeyoonLee committed:
added Aditi changes

aditid committed:
new files and repo name changes

TaeyoonLee committed:
added import for data scraping

TaeyoonLee committed:
small changes

TaeyoonLee committed:
fixed latitude longitude mix up

TaeyoonLee committed:
added prepData file

aditid committed:
drug crimes only

TaeyoonLee committed:
oops

tyao123 committed:
Edited provenance

tyao123 committed:
Merge branch 'master' of https://github.com/tyao122/course-2016-fal-proj

 Please enter a commit message to explain why this merge is necessary,

tyao123 committed:
Fixed names and added Provenance

tyao123 committed:
Added provenance changes

aditid committed:
optimization

aditid committed:
optimization

TaeyoonLee committed:
fixed db naming mix ups

TaeyoonLee committed:
update README file

TaeyoonLee committed:
fix small spacing issue

aditid committed:
optimize done

tyao123 committed:
Added provenance and limit 300000

aditid committed:
file runner

TaeyoonLee committed:
removed entries with 0 lat/long values

aditid committed:
correction to map reduce

aditid committed:
if statement correction

benli95 committed:
Commiting density creator

aditid committed:
changes

aditid committed:
Merge branch 'master' of https://github.com/tyao123/course-2016-fal-proj

aditid committed:
commit

tyao123 committed:
Fixed Provenance

TaeyoonLee committed:
added to README

TaeyoonLee committed:
Merge branch 'master' of https://github.com/tyao123/course-2016-fal-proj

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

aditid committed:
some analysis

aditid committed:
Merge branch 'master' of https://github.com/tyao123/course-2016-fal-proj

tyao123 committed:
Added provenance to prepData2

aditid committed:
images

aditid committed:
commit

aditid committed:
Merge branch 'master' of https://github.com/tyao123/course-2016-fal-proj

aditid committed:
commit

aditid committed:
img test

benli95 committed:
Putting in Prov edit to to get Data, Trial modes and scrape path comment

benli95 committed:
Putting in Prov edit to to get Data, Trial modes and scrape path comment

aditid committed:
readme

aditid committed:
commit

aditid committed:
commit

aditid committed:
helpful notes

tyao123 committed:
Added Provenance

benli95 committed:
Added trial mode for getData and editted the readme a bit

tyao123 committed:
Final fixes to README

tyao123 committed:
Actual final update to README

aditid committed:
optimization data code

aditid committed:
map reduce

aditid committed:
batch size added

aditid committed:
batch size added

lapets commented:
Could you resolve the merge conflict with `teayoon_tyao/README.md`? Then I can merge this into the shared repository.

aditid committed:
averagin function

aditid committed:
averagin function

TaeyoonLee committed:
edit visualizatoin

aditid committed:
commit

aditid committed:
Merge branch 'master' of https://github.com/tyao123/course-2016-fal-proj

tyao123 committed:
Added new readme for merging issue

tyao123 commented:
Hi Professor Lapets,

I believe we fixed the merging issue with the read me file. We just added
back the teayoon_tyao folder that we used in project 1 and included the
updated read me inside of it. Let us know if the merging issues are fixed.

Thanks

On Sun, Dec 4, 2016 at 11:57 AM, Andrei Lapets <notifications@github.com>
wrote:

> Could you resolve the merge conflict with teayoon_tyao/README.md? Then I
> can merge this into the shared repository.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Data-Mechanics/course-2016-fal-proj/pull/56#issuecomment-264715572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOHGRb1YhifkGGut8_0tbLJSF9C0Okj-ks5rEvDigaJpZM4KyG9V>
> .
>
